starter-code/css/main.css

```diff
@@ -4,7 +4,52 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
+ 
+
 }
+.main {
+
+  display: flex;
+  flex-direction: row;
+  padding: 100px;
+  width: 1000px;
+  align-items: center;
+  justify-content: center;
+  
+
+}
```

Comment:
No es buena idea darle anchos fijos a elementos muy grandes, sobre todo si no tienes preparados otros anchos para otros tipos de pantalla. Si mi pantalla fuera de 4k tu main ocuparía solo 1/4!

![](https://pics.me.me/thumb_is-this-a-picture-for-ants-memegenerator-net-is-this-a-51166932.png)

Reply:
Btw, si le das un ancho fijo asegurate de darle un margin: 0 auto; para que se quede centrado.